File: projectEuler/p6.c
#include <stdio.h>

/**
 * main - Problem 6
 *
 * Description: Sum square difference
 *
 * Return: Always 0 (Success)
*/

int main(void)
{
	int square, sqSum, sum;
	square = sqSum = sum = 0;

	for (int num = 1; num <= 100; ++num)
	{
		/* find sum of squares*/
		square = num * num;
		sqSum += square;

		/*sum of the first 100 natural numbers*/
		sum += num;
	}

	printf("The square of the sum is %d and the sum of the squares is %d\n", (sum * sum), sqSum);
	printf("The difference is %d\n", ((sum * sum) - sqSum));

	return (0);
}

File: projectEuler/p6.py
#!/usr/bin/env python3
"""
Problem 6: Sum square difference
"""

sUm = 0
sqSum = 0

for num in range(1,101):
    sqSum += num ** 2
    sUm += num

print(f"The square of the sum is {(sUm * sUm)} and the sum of the squares is {sqSum}")
print(f"The difference is {(sUm * sUm) - sqSum}")
